Packages/Tracking Preview/HandRays/Runtime/EyePinchHandRay.cs

```diff
@@ -0,0 +1,137 @@
+using Leap;
```

Comment:
This file should have copyright header and namespace as it is in the preview package

Reply:
thanks - done!